codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/protocols/parse/CborParserGenerator.kt

```diff
@@ -103,13 +103,13 @@ class CborParserGenerator(
             *codegenScope,
             "ListSymbol" to listSymbol,
         ) {
-            val deserializeMemberWritable = writable { deserializeMember(memberShape) }
+            val deserializeMemberWritable = deserializeMember(memberShape)
             if (isSparseList) {
                 rustTemplate(
                     """
                     let value = match decoder.datatype()? {
                         #{SmithyCbor}::data::Type::Null => {
-                            let _v = decoder.null()?;
+                            decoder.null()?;
```

Comment:
I wrote the original because it better conveys that we're conciously ignoring the return type; it's a common Rust idiom.